doc/dev/index.rst

```diff
@@ -502,60 +506,99 @@ from 20 minutes to three hours to complete, but it's worth the wait.
-were run) and are also reported on the `ceph-qa mailing list
+were run, i.e. via `teuthology-suite`_ or `teuthology-openstack`_), and are
+also reported on the `ceph-qa mailing list
 <http://ceph.com/resources/mailing-list-irc/>`_ for analysis.
 
-Some Ceph developers have access to the hardware running these tests
-(either bare metal or OpenStack provisioned) and are allowed to
-schedule integration tests there (the developer nick shows in the test
-results URL).
+Running integration tests
+-------------------------
 
-Ceph developers who have access to an OpenStack tenant (could be the Sepia
-OVH one or any other) can use the `ceph-workbench ceph-qa-suite`_ command
-to run integration tests and publish the results at
-http://teuthology-logs.public.ceph.com. This allows reviewers to verify
-that changes to the code base do not cause regressions, or to analyze test
-failures when they do occur.
+Some Ceph developers have access to the `Sepia lab`_ and are allowed to
+schedule integration tests there. The developer nick shows in the test
+results URL and in the first column of the Pulpito dashboard.
+
+"How can I run the integration tests in my own environment?" you may ask.
+One option is to set up a teuthology cluster on bare metal. Though this is
+a non-trivial task, it `is` possible. Here are `some notes
+<http://docs.ceph.com/teuthology/docs/LAB_SETUP.html>`_ to get you started
+if you decide to go this route.
+
+If you have access to an OpenStack tenant, you have another option:
+teuthology has an OpenStack backend, which is documented `here
+<https://github.com/dachary/teuthology/tree/openstack#openstack-backend>`_.
+While `teuthology-openstack`_ is fairly straightforward, the whole process
+is even easier if one uses `ceph-workbench ceph-qa-suite`_. This tool
+enables the developer to automate deployment of teuthology to an
+OpenStack environment and to control that cloud environment from her local
+machine. It also automates publishing of test results at
+http://teuthology-logs.public.ceph.com. 
 
 .. _`ceph-workbench ceph-qa-suite`: http://ceph-workbench.readthedocs.org/
 
-Understanding make check tests
-==============================
+Running integration tests on your code contributions and publishing the
+results allows reviewers to verify that changes to the code base do not
+cause regressions, or to analyze test failures when they do occur.
 
-Principles of make check tests, where to find the results, how to interpret
-them, how to find the corresponding source code, how to write a make check
-test.
+teuthology-suite
+----------------
 
-Understanding integration tests
-===============================
+Every teuthology cluster, whether bare-metal or cloud-provisioned, has a
+so-called "teuthology node" from which tests suites are triggered using the
+``teuthology-suite`` command. Even if `teuthology-openstack`_ or
+`ceph-workbench ceph-qa-suite`_ is used, ``teuthology-suite`` is run
+behind the scenes.
```

Comment:
I would skip the teuthology-openstack / ceph-workbench part so the reader is focused on teuthology-suite.